src/app/icd/client/DefaultICDClientStorage.cpp

```diff
@@ -235,16 +235,22 @@ CHIP_ERROR DefaultICDClientStorage::Load(FabricIndex fabricIndex, std::vector<IC
         ICDClientInfo clientInfo;
         TLV::TLVType ICDClientInfoType;
         NodeId nodeId;
+        NodeId checkInNodeId;
         FabricIndex fabric;
         ReturnErrorOnFailure(reader.EnterContainer(ICDClientInfoType));
         // Peer Node ID
         ReturnErrorOnFailure(reader.Next(TLV::ContextTag(ClientInfoTag::kPeerNodeId)));
         ReturnErrorOnFailure(reader.Get(nodeId));
 
+        ReturnErrorOnFailure(reader.Next(TLV::ContextTag(ClientInfoTag::kCheckInNodeId)));
+        ReturnErrorOnFailure(reader.Get(checkInNodeId));
```

Comment:
This is a backwards-incompatible change to the storage format.  I assume that's OK?

Reply:
yes, it should be ok considering 1.4 has not yet released and this is internal change.